README.md

```diff
@@ -1,2 +1,22 @@
 # hw01
 Repository for homework 01
+
+# This is a brief description of Elsy Gonzalez
+
+My name is Elsy Gonzalez, I am a third year Ph.D student in **Political Science** with a specialization in *International Relations*.
+
+I have a BA degree in Political Science and International Relations from the *Centro de Investigación y Docencia Economicas* in Mexico City, Mexico.
+
+## Interests
+
+My substantive interests are the following:
+
+1. International recognition of statehood
```

Comment:
With enumerated lists, you don't need to manually specify the value. Markdown will render it based on the number of previous items. So this list could be written

```
1. International recognition of statehood
1. Recognition of secessionist and irredentist movements
1. Diaspora lobbying
    a. I am specifically interested in diaspora lobbying efforts for the purpose of garnering recognition for the independence of a secessionist/irredentist movement.
```

1. International recognition of statehood
1. Recognition of secessionist and irredentist movements
1. Diaspora lobbying
    a. I am specifically interested in diaspora lobbying efforts for the purpose of garnering recognition for the independence of a secessionist/irredentist movement.